Gemfile

```diff
@@ -17,3 +17,4 @@ gem 'json', '~> 1.8.2'
 gem 'git', '~> 1.2.6'
 gem 'trollop', '~> 2.0'
 gem 'coderay', '~> 1.1.0'
+gem 'rb-inotify', '~> 0.9'
```

Comment:
I can build without it using docker https://github.com/infinispan/infinispan.github.io/tree/develop/bin/docker. Is it failing on the release server?